get_pdfs.go

```diff
@@ -25,34 +27,47 @@ func initConfig() {
 	if err != nil {
 		log.Fatal()
 	}
+
+	clientCSV = client.Client{
+		APIKey:     CONFIG.PDFTABLES_API_KEY,
+		HTTPClient: http.DefaultClient,
+	}
+}
+
+func folderCheck() {
+	if _, err := os.Stat(CONFIG.RAW_PDF_PATH); os.IsNotExist(err) {
+		os.MkdirAll(CONFIG.RAW_PDF_PATH, 0700)
+	}
+	if _, err := os.Stat(CONFIG.RAW_CSV_PATH); os.IsNotExist(err) {
+		os.MkdirAll(CONFIG.RAW_CSV_PATH, 0700)
+	}
 }
 
 func main() {
 	initConfig()
+	folderCheck()
 	//Go Pool does the trick!!!
 	p := pool.NewPool(CONFIG.QUEUE_SIZE, CONFIG.WORKER_NUM)
 	p.Start()
 	for i := CONFIG.START; i <= CONFIG.END; i++ {
 		p.Add(FileDownloader{
```

Comment:
hmmm, I'm thinking we should probably decouple FileDownloader from FileConverter 🤔

is there a way of returning the file that file name that has been downloaded so that we can call a FileConverter on it?

If not, then let's do another for loop after this of only the files that are saved and run the converter through it, or at least the first [however many free ones are available]

Reply:
That will probably require us to scan the folder then convert all the PDFs in the folder on its own separate pool.

Reply:
ok, then let's call the method DownloadAndConvertMSEPdfs

Reply:
I already made it a closure, I should revert?

Reply:
no I meant the Struct, it should be called MSEFileDownloaderAndCoverter